plugins/scaffolder-react/src/next/components/ScaffolderPageContextMenu/ScaffolderPageContextMenu.tsx

```diff
@@ -29,38 +29,35 @@ import List from '@material-ui/icons/List';
 import MoreVert from '@material-ui/icons/MoreVert';
 import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import {
-  actionsRouteRef,
-  editRouteRef,
-  scaffolderListTaskRouteRef,
-} from '../../routes';
 
 const useStyles = makeStyles((theme: BackstageTheme) => ({
   button: {
     color: theme.page.fontColor,
   },
 }));
 
+/**
+ * @alpha
+ */
 export type ScaffolderPageContextMenuProps = {
-  editor?: boolean;
-  actions?: boolean;
-  tasks?: boolean;
+  editor?: RouteFunc<AnyParams>;
+  actions?: RouteFunc<AnyParams>;
+  tasks?: RouteFunc<AnyParams>;
 };
 
-export function ContextMenu(props: ScaffolderPageContextMenuProps) {
+/**
+ * @alpha
+ */
+export function ScaffolderPageContextMenu(
+  props: ScaffolderPageContextMenuProps,
```

Comment:
Thinking about these props a little more, thinking that it's nicer to just provide callbacks here rather than `routeRefs`. It provides much more flexibility so that you can do other things rather than just navigate.

Also not sure that passing `routeRefs` are a pattern that we've used previously.